cmd/seed.go

```diff
@@ -0,0 +1,33 @@
+package cmd
+
+import (
+	"context"
+
+	"github.com/spf13/cobra"
+
+	"go.infratographer.com/identity-api/internal/config"
+	"go.infratographer.com/identity-api/internal/storage"
+)
+
+func init() {
+	rootCmd.AddCommand(seedDatabaseCmd)
+}
+
+var seedDatabaseCmd = &cobra.Command{
+	Use:   "seed-database",
+	Short: "seeds identity-api database",
```

Comment:
Seeds it with what? Let's make it quite clear to readers what's being seeded so this is more usable.